libs/community/langchain_community/chat_message_histories/singlestoredb.py

```diff
@@ -153,7 +153,7 @@ def __init__(
             self.connection_kwargs["conn_attrs"] = dict()
 
         self.connection_kwargs["conn_attrs"]["_connector_name"] = "langchain python sdk"
-        self.connection_kwargs["conn_attrs"]["_connector_version"] = "1.0.1"
+        self.connection_kwargs["conn_attrs"]["_connector_version"] = "2.1.0"
```

Comment:
not necessarily opposed to this but wondering if it can break user workflows

Reply:
It cannot break anything, the library reports to the database engine the version of the connector code. We can put any number there, making sure that it is incremental.

Reply:
"2.1.0" is the version we use in the vector store, so I put it here for consistency.